client/macos/Podfile.lock

```diff
@@ -57,4 +57,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 353c8bcc5d5b0994e508d035b5431cfe18c1dea7
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.12.1
```

Comment:
@FeodorFitsner dont you think we should prevent the constant update of this cocoapods version? (ex: add to gitignore?)

Reply:
Yeah, I'm pretty tired of that.